src/devtools/client/inspector/markup/components/TextNode.tsx

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from "react";
 
 import NodeConstants from "devtools/shared/dom-node-constants";
 import { assert } from "protocol/utils";
+import { reIsNotWhiteSpace } from "ui/suspense/nodeCaches";
```

Comment:
What is this? What is "re"? I'm guessing "regular expression"?

Can we name this something more semantically meaningful?

Reply:
I've always prefixed my regex variable names with `re`, yeah. Longstanding habit.  Better suggestions?

Reply:
Whoa! "re" means...nothing to me, except as a prefix meaning "again"

Maybe "regex" ?